simulator/A-NC/A_NC.py

```diff
@@ -0,0 +1,39 @@
+from utils import *
+
+def run(train_loader, test_loader, comm_matrix, num_rounds, epochs, num_clients,
+        net='net', optimizer='sgd', lr=0.1):
+    """
+    Runs a decentralized optimization algorithme for the given learning rate for
+    a number of rounds, over some network. Outputs the accuracies and returns them.
+
+    Params:
+        train_loader (array): the list of all train datasets, one per client
+        test_loader (array): the list of test datasets, one per client
+        comm_matrix (numpy.array): the communication matric modeling the network
+        num_rounds (int): the number of data exchanges between nodes
+        epochs (int): the number of optimization steps between each communication (minimum 1)
```

Comment:
the `epoch` here is confusing. is it correspond to the number of local update steps?